Demo/Common/IntegrationExamples/Protocols/DropInExampleProtocol.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import AdyenNetworking
+import AdyenSession
+
+internal protocol DropInExampleProtocol: APIClientAware {
```

Comment:
I think the only difference between this and `ComponentExampleProtocol` would be `palApiClient` ?

Reply:
Technically it is backend's job to do PAL calls. Maybe we can simply combine `palApiClient` and `apiClient` ?
Or even move `palApiClient` out of the scope of "simple DropIn" example?

Reply:
I have removed ```palApiClient``` from dropIn.